.env.example

```diff
@@ -21,6 +21,7 @@ NEXT_PUBLIC_CAPTCHA_SITE_KEY=<captcha sitekey>
 
 NEXT_PUBLIC_MODERATION_URL=<moderation service url>
 # e.g. https://moderation.subsocial.network/graphql
+NEXT_WEB3AUTH_CLIENT_ID=BLxj7JujmtinIZnqY0f6NL_tZaQGwiktzxLgJJMzOEKhJIKANng6G-5UbxmkUwrflLNSe23_ZTG4f3krwweqRFA
```

Comment:
For this, it should also be removed and changed into just explanation, but for now, this one is okay, let's change it after this is good to go to be merged